client/my-sites/site-settings/disconnect-site/why/disconnect-reasons-menu.jsx

```diff
@@ -0,0 +1,99 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import SelectDropdown from 'components/select-dropdown';
+import PaginationFlow from './pagination-flow';
+import { isJetpackSite } from 'state/sites/selectors';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+import DisconnectReasonsSuggestion from './disconnect-reasons-suggestion';
+
+class DisconnectReasonsMenu extends Component {
+	constructor( props ) {
+		super( props );
+		const initialState = {
+			childSelected: 'missingFeature',
+			compactButtons: false,
+			renderFull: false
+		};
+
+		this.state = initialState;
+		this._logReason = this._logReason.bind( this );
+	}
+
+	displayFull = ( option ) => {
+		return (
+			<DisconnectReasonsSuggestion
+				optionSelected={ option }
+				compactButtons={ this.state.compactButtons } />
+		);
+	}
+
+	_logReason = ( option ) => {
+		this.setState( {
+			childSelected: option.value,
+			renderFull: true
+		} );
+	}
+
+	render() {
+		const {
+			translate,
+			siteSlug
+				} = this.props;
+		const {
+			childSelected,
+			compactButtons,
+			renderFull
+		} = this.state;
+
+		const textShareWhy = translate( 'Would you mind sharing why you want to' +
+			' disconnect ' ) + `${ siteSlug }` + translate( ' from Wordpress.com ' );
+
+		const options = [
+			{ value: 'missingFeature', label: 'Jetpack is missing a feature' },
+			{ value: 'reason2', label: 'I\'m going to use WordPress somewhere else' },
```

Comment:
Let's make sure we use the correct values here. This is important information that will help us improve our services, so we'll be tracking it accurately. Let's try to reuse all of the current options in the old questionnaire - for example `reason2` should be `otherWordPress` instead.